action-sonos.py

```diff
@@ -38,8 +38,10 @@
 MOPIDY_HOST = HOSTNAME
 
 # Logging config
-logging.basicConfig(level=logging.INFO)
-
+if HERMES_HOST != "localhost":
```

Comment:
Same here, maybe we could provide a flag, in the `sys.argv` or in the `config.ini`

Reply:
I would go for `config.ini`